tests/cupy_tests/test_cudnn.py

```diff
@@ -37,6 +42,27 @@ def test_activation_backward(self):
         cupy.cudnn.activation_backward(self.x, self.y, self.g, self.mode)
 
 
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64],
+    'mode': coef_modes,
+}))
+@unittest.skipUnless(cudnn_enabled, 'cuDNN is not available')
```

Comment:
Please fix test for cuDNN v4 environemt.

Reply:
Ok, I fixed it.